src/Views/FirmwareView.vala

```diff
@@ -216,7 +216,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
     private void header_rows (Widgets.FirmwareUpdateRow row1, Widgets.FirmwareUpdateRow? row2) {
         if (row2 == null && row1.is_updatable) {
             var header = new FirmwareHeaderRow (
-                dngettext (GETTEXT_PACKAGE, "%u Update Available", "%u Updates Available", num_updates).printf (num_updates)
+                dngettext (GETTEXT_PACKAGE, "%u new firmware available", "%u new firmwares available", num_updates).printf (num_updates)
```

Comment:
This sounds much less natural. In US English I don't believe that firmware uses discrete units. It sounds like saying "10 waters". This also should be title case since it is a row header

Reply:
I was going with the concept of "ware" many "wares", but that is quirky.